### DISCUSSION TITLE
Synchronizing local SQLite Databases between different time zones

### DISCUSSION THREAD

**Discussion by ricardoamm** [Mimetis/Dotmim.Sync#1285] [General]
Hello,

I'm facing some strange issues, where some local SqlLite Providers are not receiving changes for the same scope.
Could the fact that they are in different time zones create any time of problem due diferent timestamps?

Regards

**Comment by Mimetis** [comment]
The timestamp value calculated by **SQLite** every time a record is saved in the tracking table is basically calculated using this SQL statement:

``` sql
Select replace(strftime('%Y%m%d%H%M%f', 'now'), '.', '')
```

In **SQLite**, `Datetime('now') `is basically returning the **UTC Datetime**.

So it should not be a problem when regarding different timezones.

**Reply by ricardoamm** [reply]
I'm having the following error in clients two often:

Error: Client database is out of date. Last client sync timestamp:376256. Last server cleanup metadata:376259 Try to make a Reinitialize sync.

**Reply by Mimetis** [reply]
On the server side, are you using **Sql Server Change Tracking** ?
If yes, what is your retention period parameter ?

**Reply by ricardoamm** [reply]
yes, 14 days

**Reply by Mimetis** [reply]
Can it be the cause of this out of date issue ?

**Reply by ricardoamm** [reply]
The error happened less than a day after the previous successful sync.

**Reply by Mimetis** [reply]
Hum, that's weird ....
It's only happens from some specific clients or it can happens to any client ?
Are you doing specific things that can help to debug this issue ?
